src/client/Microsoft.Identity.Client/Cache/Items/MsalRefreshTokenCacheItem.cs

```diff
@@ -53,6 +53,7 @@ internal MsalRefreshTokenCacheItem()
         public string FamilyId { get; set; }
 
         internal string UserAssertionHash { get; set; }
+        internal string OboCacheKey { get; set; }
```

Comment:
As above, try to reuse UserAssertionHash. Maybe rename "UserAssertionHash" variable name to "OBOSessionKey" (but do not change the underlying JSON).